### ISSUE TITLE
Add simplifications for elm-explorations/test

### ISSUE THREAD

**Issue by jfmengels** [jfmengels/elm-review-simplify#108]
So far Simplify only supports core Elm packages (in part because they're really stable), but I think the `elm-explorations` ones could warrant being handled as well.

Here are some I could think of, but I'm sure a pass over the API could find more.

## Expect

```elm
-- To help with transitioning from v1 to v2
Expect.true string value
--> foo
-->  |> Expect.equal True
-->  |> Expect.onFail string

Expect.false string value
--> value
-->  |> Expect.equal False
-->  |> Expect.onFail string
```

## Fuzz

```elm
Fuzz.filter (always True) a
--> a

Fuzz.filter (always False) a
--> ???

Fuzz.map identity a
--> a

Fuzz.andThen Fuzz.constant a
--> a

Fuzz.andThen (\b -> Fuzz.constant (f b)) a
--> Fuzz.map (\b -> f b) a
```

- Something with `Fuzz.constants`?
- Something about `Fuzz.lazy` being useless?

**Comment by lue-bird** [comment]
Just if you didn't know: `Expect.true` and `Expect.false` were removed in 2.0.0, see https://github.com/elm-explorations/test/blob/2.1.1/CHANGELOG.md#4-expecttrue-and-expectfalse-removal

**Comment by jfmengels** [comment]
Yes, which is why I wrote "To help with transitioning from v1 to v2" 👍
But yeah, whether that's worth the effort I don't know 😁
